modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/DeployService.java

```diff
@@ -823,4 +833,194 @@ public DeployServiceEntity getDeployServiceEntity(UUID id) {
         }
         return deployServiceEntity;
     }
+
+    /**
+     * Start the service by the deployed service id.
+     *
+     * @param id service id.
+     * @return deployedService.
+     */
+    public DeployedService startService(UUID id) {
```

Comment:
in our discussion over chat and also in the userstory.. i have requested to make them async method if needed.

Reply:
@swaroopar Hi swaroop,The API batchStartServers, batchStopServers, and batchRebootServers of Huawei ECS are themselves asynchronous. I don’t think we need to make this method asynchronous.